### PR TITLE
Assignment 10

### PR DESCRIPTION
**Pull request by AndrewPerel**

### CODE REVIEW

File: Assignments/Assignment 10/Assignment 10.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        request.addEventListener("load", result);
+        request.send();
+
+        function result(e) {
+            if (e.target.status >= 400) {
+                document.getElementById("status").innerHTML = `${e.target.status}`;
+                return;
+            }
+
+            const fullResult = String(JSON.parse(e.target.responseText).datetime);
```
Comment (akennis) [inline Assignments/Assignment 10/Assignment 10.js:20]: You should not convert `responseText` which is a string to an object only to go back to a string - instead you should pull data out of the object based on its keys.

File: Assignments/Assignment 10/Assignment 10.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+function main() {
+    document.getElementById("enterBTN").addEventListener("click", getTime);
+
+    function getTime(e) {
+        const timeZone = document.getElementById("timeZone").value;
+        const URL = `http://worldtimeapi.org/api/timezone/${timeZone}`;
+
+        const request = new XMLHttpRequest();
+
+        request.open("GET", URL, true);
```
Comment (akennis) [inline Assignments/Assignment 10/Assignment 10.js:10]: Please also play with setting a timeout and adjust that to be higher AND also lower than the API's actual performance in order to see timeouts occur and communicate those to the user.

**Review by akennis — COMMENTED** [review]
Good work so far - see my comments - 85% as-is.

File: Assignments/Assignment 10/Assignment 10.js (hunk 1 above)
Comment (AndrewPerel) [reply Assignments/Assignment 10/Assignment 10.js]: Resolved both comments, please check

**Pull request merged**